packages/react-topology/src/components/groups/DefaultGroupExpanded.tsx

```diff
@@ -121,6 +121,9 @@ const DefaultGroupExpanded: React.FunctionComponent<DefaultGroupExpandedProps> =
   const hullPadding = React.useCallback((point: PointWithSize | PointTuple) => (point[2] || 0) + padding, [padding]);
 
   const points: (PointWithSize | PointTuple)[] = React.useMemo(() => {
+    if (droppable) {
+      return prevPoints.current;
+    }
```

Comment:
If `prevPoints.current` hasn't been defined, you should still run this function completely.